example_dht_sensor.py

```diff
@@ -0,0 +1,24 @@
+from sensors.Dht22 import Dht22 
+
+
+# uid: int, gpio_in: int, timeout : int, model : str 
+
+GPIO_IN = 4 # DHT22 gpio data in
+MODEL = 'DHT22' # Not needed
+UUID = 5001 # Some random fake uuid
+
+
+def main():
+    dht22_obj = Dht22(UUID, GPIO_IN, MODEL)
+    try:
+        dht22_obj.connect_to_sensor()
+        dht22_obj.take_reading()
+    except SensorReadError:
+        reading_data = {"error": "sensor error"}
```

Comment:
rename this variable to sensor_readings so that the error is output

Reply:
Thanks for pointing out, should have used the same variable